<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>463</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Youtube Downloads</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>KacstOne</family>
         <pointsize>28</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>YOUTUBE</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>10</y>
         <width>311</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>KacstOne</family>
         <pointsize>28</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string> DOWNLOADS</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(111, 111, 111);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>280</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(99, 0, 0);</string>
       </property>
       <property name="text">
        <string>DOWNLOAD</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>200</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>URL:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>50</y>
         <width>341</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>KacstOne</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>true</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>SELECIONE UMA OPÇÃO</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="option_video">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>120</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>VIDEO</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="option_playlist">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>120</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>PLAYLIST</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="option_audio">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>120</y>
         <width>104</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>ÁUDIO</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>190</y>
         <width>471</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border-style:outset;
border-width: 1px;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Digite a url</string>
       </property>
      </widget>
      <zorder>label_3</zorder>
      <zorder>pushButton</zorder>
      <zorder>label_2</zorder>
      <zorder>option_video</zorder>
      <zorder>option_playlist</zorder>
      <zorder>option_audio</zorder>
      <zorder>lineEdit_2</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
